### ISSUE TITLE
HighCharts not loading TimeLine when there is more than 1000 Events

### ISSUE THREAD

**Issue by mcvic1rj** [gophish/gophish#1015]
What version of Gophish are you using?: 4.0 - but this issue should be present in 5.0 as well

Brief description of the issue: HighCharts Timeline will only load 1000 points by default. Assuming a phish where everyone Opens, Clicks and Submits credentials 1 time, the max phish size for the chart to render is 249 people

What are you expecting to see happen? : The timeline chart to render when phishing <1000 people

What are you seeing happen? A blank Timeline chart area

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Generate a phish to 1001 addresses, send email out
 * Step 2 - View campaign results, see there is no chart
 
Steps to fix this:
In the following [code block](https://github.com/gophish/gophish/blob/f21536da7c4c654e3ad531ba1ad9fa21870893f4/static/js/src/app/campaign_results.js#L398) (And the related min.js file) add the `turboThreshold` key. But it would need to be decided on what to set this to. By default it is 1000. Setting it to 0 will be "Unlimited" but that could cause page loading issues with large campaigns.

Ex.
```json
plotOptions:{
            series: {
                turboThreshold: 5000,
                marker: {
                    enabled: true,
                    symbol: 'circle',
                    radius: 3
        },
...
````

**Comment by jordan-wright** [comment]
Hi @mcvic1rj, thanks for reaching out! I'm pretty sure this was fixed in 0.5.0 in https://github.com/gophish/gophish/commit/e785af5c0aff104318628b01b9fcf4a15bdc7d44#diff-851cc3a2890b735d948e7137cfb8e07eR391. We wound up just setting it to "0" in the meantime until we can find a better solution.

Hope this helps! Let me know if you continue to run into issues.

**Comment by mcvic1rj** [comment]
@jordan-wright Looks I did not search well enough. I will close out this issue! 

-
Ryan
